src/Service/PaymentMethodsService.php

```diff
@@ -156,10 +177,6 @@ public function getSplitStreetAddressHouseNumber(string $address): array
      */
     private function buildPaymentMethodsRequestData(SalesChannelContext $context, $orderId = '')
     {
-        if (is_null($context->getCustomer())) {
-            return [];
-        }
-
```

Comment:
removing this allows us to make a paymentMethods call from the shopping cart where there is no logged in/guest customer